metcdv3/balancer_test.go

```diff
@@ -24,14 +24,15 @@ func TestFairBalancer(t *testing.T) {
 		return false // never done
```

Comment:
can you create a ticket to add a testcase for the canary/non-canary use case?   Doesn't have to be on this PR.